components/Dialogs/tests/unit/MDCAlertControllerTests.m

```diff
@@ -273,4 +273,21 @@ - (void)testAccessibilityIdentifiersAppliesToAlertControllerViewButtons {
   XCTAssertEqualObjects(button2.accessibilityIdentifier, @"A");
 }
 
+- (void)testCornerRadius {
```

Comment:
Please add a test to verify that the default radius is 0 since it is documented in the public header.